owncloudApp/src/main/java/com/owncloud/android/presentation/transfers/TransferListComposeFragment.kt

```diff
@@ -0,0 +1,532 @@
+                                .height(25.dp),
+                            painter = painterResource(id = imageResource),
+                            tint = colorResource(id = R.color.half_black),
+                            contentDescription = "Button"
+                        )
+                    }
+                }
+            }
+        }
+        Divider(
+            modifier = Modifier
+                .fillMaxWidth(),
+            color = colorResource(id = R.color.filelist_icon_background),
+            thickness = 1.dp
+        )
+    }
+
+    @Composable
+    fun SpacePathLine(space: OCSpace, parentPath: String) {
+        val spaceName: String
+        val spaceImage: Int
+
+        if (space.isPersonal) {
+            spaceName = "Personal"
+            spaceImage = R.drawable.ic_folder
+        } else {
+            spaceName = space.name
+            spaceImage = R.drawable.ic_spaces
+        }
+
+
+        Row(
+            modifier = Modifier
+                .fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                modifier = Modifier
+                    .layoutId("space_icon")
+                    .width(15.dp)
+                    .height(15.dp)
+                    .padding(end = 2.dp),
+                painter = painterResource(id = spaceImage),
+                tint = colorResource(id = R.color.list_item_lastmod_and_filesize_text),
+                contentDescription = "Space Icon"
+            )
+            Text(
+                modifier = Modifier
+                    .layoutId("space_name")
+                    .padding(end = 8.dp),
+                color = colorResource(id = R.color.list_item_lastmod_and_filesize_text),
+                fontWeight = FontWeight.Bold,
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
+                fontSize = 12.sp,
+                text = spaceName
+            )
+            Text(
+                modifier = Modifier
+                    .layoutId("path"),
+                color = colorResource(id = R.color.list_item_lastmod_and_filesize_text),
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
+                fontSize = 12.sp,
+                text = parentPath
+            )
+        }
+    }
+
+    @Composable
+    fun EmptyList() {
+        Column(
+            modifier = Modifier
+                .padding(top = 8.dp, bottom = 8.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center
+        ) {
+            Icon(
+                modifier = Modifier
+                    .layoutId("list_empty_dataset_icon")
+                    .width(72.dp)
+                    .height(72.dp),
+                tint = colorResource(id = R.color.grey),
+                painter = painterResource(id = R.drawable.ic_uploads),
+                contentDescription = "Empty dataset Icon"
```

Comment:
Content descriptions should go to the `strings.xml` file as well, so that we can provide accessibility in different languages

Reply:
Changed ✅